src/utils/endToEndId.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace StarkInfra\Utils;
+
+
+class EndToEndId
+{
+    public static function create($ispb)
+    {
+        $source = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
+        $endToEndId = "E" . $ispb . date('YmdHi');
+        for ($i = 0; $i <= 10; $i++) {
+            $endToEndId .= $source[rand(0, (strlen($source)-1))];
```

Comment:
pode ser excesso de otimização, mas faria assim pra evitar calcular isso em todo loop:
$randLength = strlen($source) - 1;
for ($i = 0; $i <= 10; $i++) {
            $endToEndId .= $source[rand(0, $randLength)];